packs/data/feats.db/elf-atavism.json

```diff
@@ -38,6 +38,11 @@
                 "flag": "heritage",
                 "key": "ChoiceSet",
                 "prompt": "PF2E.SpecificRule.ElfAtavism.Prompt"
+            },
+            {
+                "key": "GrantItem",
+                "allowDuplicate": false,
+                "uuid": "{item|flags.pf2e.rulesSelections.heritage}"
```

Comment:
This doesn't replace the existing heritage?

Reply:
It don't!

Reply:
https://github.com/foundryvtt/pf2e/pull/6377